### PR TITLE
JBIDE-22102 - add astools n&n

### PR DESCRIPTION
**Pull request by robstryker**

### CODE REVIEW

File: documentation/whatsnew/server/server-news-4.3.1.Final.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+In addition, several schema had small updates, bugfixes, and documentation changes. All of these will be available in the XML Catalog now. 
+
+related_jira::JBIDE-20529[Include updated WildFly 10 schema in catalog]
+
+== Publishing errors on Windows now fixed. 
+
+Windows often locks files when it shouldn't, and this causes a big problem for filesystem-based
+deployments that look to make changes on-the-fly.  We've managed to improve our integration 
+with the WildFly application server, and to make sure we won't make changes to files while they're still
```
Comment (fbricon) [inline documentation/whatsnew/server/server-news-4.3.1.Final.adoc]: , and to make sure we won't make changes... 

I'm expecting something here

Comment (robstryker) [inline documentation/whatsnew/server/server-news-4.3.1.Final.adoc]: ?   huh?  seems like a full sentence to me?

**Comment by alexeykazakov** [comment]
Rebased and pushed to master. Not in production yet.

**Comment by alexeykazakov** [comment]
Pushed to production.

**Pull request closed without merging**